tests/publish.rs

```diff
@@ -61,6 +63,9 @@ async fn test_publish_and_subscribe() {
     // Waiting for the subscriber to receive data
     task::sleep(time::Duration::from_millis(1000)).await;
 
+    // Compare the result
+    assert_eq!(*verified_data.lock().unwrap(), target_data);
```

Comment:
Yup, makes sense. Good way to ensure we're passing the test :slightly_smiling_face: